### ISSUE TITLE
how to preserve a better identity?

### ISSUE THREAD

**Issue by loboere** [eeskimez/emotalkingface#2]
When I use an image of mine it is distorted a lot and does not look like the original face, is there any parameter to improve the identity?

**Comment by ujjawalcse** [comment]
yaa Sure,
Here is the `results.zip` file which contains the `condition.png` (reference image) and the output videos in different expressions.

[results.zip](https://github.com/eeskimez/emotalkingface/files/9523821/results.zip)

**Comment by eeskimez** [comment]
Are you using our pre-trained model? If so, it is trained on the CREMA-D dataset, which has limited samples/data distribution, and it might not generalize well to images outside of CREMA-D. Therefore, if you want it to generalize well, you might need a more extensive dataset, which is hard to find since we require emotion labels. You can try omitting emotion labels and use the LRW dataset for better generalization.

**Comment by yzyouzhang** [comment]
I checked your output videos and find that the speech are much longer than those in our training data (~2s). The quality of the first two seconds are reasonable. This might suggest the generalization ability needs to be improved as Emre said.

For the dataset, there is a new emotional talking face dataset released after our publication, called MEAD. Feel free to try to train our model on that dataset. I am also interested in the results.

**Comment by ujjawalcse** [comment]
Thanks @eeskimez and @yzyouzhang for clarification.
